dotnetv3/S3/scenarios/TransferUtilityBasics/TransferUtilityBasics/TransferBasics.cs

```diff
@@ -50,9 +68,11 @@
 
 PressEnter();
 
-
 // Download a single file from an S3 bucket.
-var keyName = "FileToDownload.docx";
+DisplayTitle("Download a single file");
```

Comment:
Missing punctuation?

Reply:
That was intended as a header. I'm leaving it for now pending Liz's review.

Reply:
If this text functions as a title, no ending punctuation is required.

Reply:
Ah, got it. Thanks!